README.rst

```diff
@@ -0,0 +1,484 @@
+
+
+The output of these commands tells us enough about each process to
+construct a Sprocket ".pipe" file that defines a processing pipeline.
+In particular we'll need to know how to configure each process (the
+"Configuration") and how they can be hooked together (the input and
+output "Ports").
+
+KWIVER comes with a sample
+[sprokit/pipelines/number_flow.pipe](sprokit/pipelines/number_flow.pipe)
+file that configures and connects the pipeline so that the `numbers`
+process will generate a set of integers from 1 to 99 and the
+`print_number` process will write those to a file called
+`numbers.txt`.  Of particular interest is the section at the end of
+the file that actually "hooks up" the pipeline.
+
+To run the pipeline, we'll use the Sprokit `pipeline_runner` command::
+
+    $ pipeline_runner -p </path/to/kwiver/source>/sprokit/pipelines/number_flow.pipe
+
+After the pipeline completes, you should find a file, `numbers.txt`, in your working directory.
+
+
+Python Processes
+----------------
+
+One of KWIVER's great strengths (as provided by sprokit) is the
+ability to create hybrid pipelines which combine C++ and Python
+processes in the same pipeline.  This greatly facilitates prototyping
+complex processing pipelines.  To test this out we'll still use the
+`numbers` process, but we'll use a Python version of the
+`print_number` process called `kw_print_number_process` the code for
+which can be seen in
+[sprokit/processes/python/kw_print_number_process.py](sprokit/processes/python/kw_print_number_process.py).
+As usual, we can lean about this process with the following command::
+
+    $ plugin_explorer --process --type kw_print_number_process -d --config
+
+	Process type: kw_print_number_process
+	  Description: A Simple Kwiver Test Process
+	  Properties: _no_reentrant, _python
+	  Configuration:
+	    Name       : output
+	    Default    : .
+	    Description: The path for the output file.
+	    Tunable    : no
+
+	  Input ports:
+	    Name       : input
+	    Type       : integer
+	    Flags      : _required
+	    Description: Where numbers are read from.
+
+	  Output ports:
+
+As you can see, the process is very similar to the C++ `print_number`
+process.  As a result, the [".pipe" file is very
+similar](sprokit/pipelines/number_flow_python.pipe).
+
+In order to get around limitations imposed by the Python Global
+Interpreter Lock, we'll use a different Sprokit scheduler for this
+pipeline.  The `pythread_per_process` scheduler which does essentially
+what it says: it creates a Python thread for every process in the
+pipeline::
+
+	pipeline_runner -S pythread_per_process -p </path/to/kwiver/source>/sprokit/pipelines/number_flow_python.pipe>
+
+As with the previous pipeline, the numbers will be written to an output file, this time `numbers_from_python.txt`
+
+
+
+vital
+=====
+
+Vital is an open source C++ collection of libraries and tools that
+supply basic types and services to the Kitware KWIVER imagery tool
+kit.
```

Comment:
I've put the overview description of Vital at the top.  I think the remaining content in this section should move into an overview of Vital document in `doc/manuals/vital`